ONSdigital/eq-terraform
# Eq 401 standardising names

**_What**_
Standardising the naming between the author and survey runner, making sure all resources have names that allow the user to know if the resource is being used for the survey runner or author application. Also where possible adding tags as these aid in locating them in the AWS console.

**How to test**
1. Checkout this branch
2. Deploy both an author and survey runner environment using terraform
3. Check everything still works

**Who can review**
Anyone apart from @warrenbailey
